brd-ios/breadwallet/src/Redesign/Views/MainSwapView.swift

```diff
@@ -171,11 +178,10 @@ class MainSwapView: FEView<MainSwapConfiguration, MainSwapViewModel> {
         
         didChangePlaces?()
 
-        // TODO: animations were breaking user interactions. disabled for now
-//        SwapCurrencyView.animateSwitchPlaces(sender: sender,
-//                                             baseSwapCurrencyView: baseSwapCurrencyView,
-//                                             termSwapCurrencyView: termSwapCurrencyView)
-//
+        SwapCurrencyView.animateSwitchPlaces(sender: sender,
```

Comment:
is there a reason this is a static method on a child view?
could MainSwapView not just handle the transforms?